### PR TITLE
Erdos Reyni - G(n,m) engine added

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
Erdős–Rényi G(n,m) random graph generation model support added

### CODE REVIEW

File: pyrgg/engines/erdos_reyni.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    :param m: number of edges
+    :type m: int
+    :param direct: directed graph flag
+    :type direct: bool
+    :return: list of dicts
+    """
+    edge_dic = {}
+    weight_list = []
+    edge_mold = []
+    TOTAL_EDGE = (n * (n - 1)) // 2
```
Comment (sepandhaghighi) [inline pyrgg/engines/erdos_reyni.py:33]: My suggestion:

```
max_edge = (n * (n - 1)) // 2
```

File: pyrgg/engines/erdos_reyni.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    if direct:
+        m = min(m, 2 * TOTAL_EDGE)
```
Comment (sepandhaghighi) [inline pyrgg/engines/erdos_reyni.py:35]: My suggestion:

```
max_edge = 2 * max_edge
m = min(m, max_edge)
edge_mold = m * [1] + (max_edge - m) * [0]
...
```

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Please review my comments 🔥

**Comment by sadrasabouri** [comment]
@sepandhaghighi would you please review this?

**Review by sepandhaghighi — APPROVED** [review]
🔥

**Pull request merged**